snuba/migrations/runner.py

```diff
@@ -60,8 +60,13 @@ class MigrationDetails(NamedTuple):
 
 
 class Runner:
-    def __init__(self) -> None:
-        migrations_cluster = get_cluster(StorageSetKey.MIGRATIONS)
+    def __init__(self, partition_id: Optional[int]) -> None:
```

Comment:
What is the plan to initialize the migrations table? 
Now the migrations table is created by this migration https://github.com/getsentry/snuba/blob/master/snuba/migrations/system_migrations/0001_migrations.py.

We will need a way to innitialize that table on a new slice when the new slice shows up in the configuration.

Reply:
in my local environment, this just sort of worked (e.g. when I created the new database (`slice_1_default`) and ran the initial migrations against it it built a new copy of the migrations table.